README.md

```diff
@@ -1 +1,33 @@
-# solution-template-repository
\ No newline at end of file
+# Release Information
+
+- **Version**:  1.0.0
+- **Certified**: Yes
+- **Publisher**: Fortinet
+- **Compatible Version**: FortiSOAR v7.6.1 and later
+
+# Overview
+
+**Unleash the Power of Threat Intelligence with Our FortiGuard Labs IOC Search Solution Pack**
+
+Empower your SOC analysts with unparalleled visibility into malicious indicators using the FortiGuard Labs IOC Search Solution Pack. This innovative solution seamlessly integrates the powerful capabilities of FortiGuard Labs into FortiSOAR, delivering actionable threat intelligence at your fingertips.
+
+- **Comprehensive IOC Insights**: Access a wealth of global cyber threat intelligence gathered from FortiGuard’s extensive network of over 3 million sensors, proprietary AI-driven web crawlers, and strategic threat-sharing alliances.
```

Comment:
from where did you get this info?

Reply:
- https://www.fortiguard.com/outbreak-alert/2023-annual-report
- https://www.fortinet.com/fortiguard/labs

Rephrased from these resources